dart/test/mocks/mock_transport.dart

```diff
@@ -17,7 +17,7 @@ class MockTransport implements Transport {
   }
 
   bool called(int calls) {
-    return calls == calls;
+    return _calls == calls;
```

Comment:
Euclid's first common notion is this: "Things which are equal to the same things are equal to each other" ^^